src/main/kotlin/com/github/hanseter/json/editor/JsonPropertiesEditor.kt

```diff
@@ -20,10 +22,11 @@ import org.controlsfx.validation.decoration.GraphicValidationDecoration
 import org.json.JSONObject
 import java.net.URI
 
-class JsonPropertiesEditor(
+class JsonPropertiesEditor @JvmOverloads constructor(
     private val readOnly: Boolean = false,
     viewOptions: ViewOptions = ViewOptions(),
     actions: List<EditorAction> = listOf(ResetToDefaultAction, ResetToNullAction),
+    private val customizationObject: CustomizationObject? = null
```

Comment:
Why do we accept null here (and make it the default), If we replace null with the default object later on anyways?